sdk/src/main/java/com/hcaptcha/sdk/HCaptchaException.java

```diff
@@ -2,36 +2,48 @@
 
 import androidx.annotation.Nullable;
 
-import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
+import lombok.NonNull;
 
 
 /**
  * A checked exception which contains an {@link HCaptchaError} id and message.
  */
 @Data
-@AllArgsConstructor
 @EqualsAndHashCode(callSuper = true)
 public class HCaptchaException extends Exception {
 
     /**
      * The hCaptcha error object
      */
-    private final HCaptchaError hCaptchaError;
+    @NonNull
+    private final HCaptchaError error;
```

Comment:
Seems unrelated to the PR, would help with the review to revert `hCaptchaError` rename to plain `error`.

Reply:
Agree, I will revert it because of breaking change as you noted below